common/current_preset.h

```diff
@@ -271,10 +271,30 @@ class CurrentPreset {
 
   void SetPreset(int preset) {
     Clear();
+    #ifdef SAVED_PRESET
+    SavePreset(preset);
+    #endif
     LOCK_SD(true);
     if (!Load(preset)) Set(preset);
     LOCK_SD(false);
   }
+
+  bool SavePreset(int preset) {
+    if (preset >= 0 && preset <= current_config->num_presets) {
```

Comment:
When is this if statement helpful?

Reply:
Removed, remnant from old way of calling that function